### ISSUE TITLE
I'm having permission issues with macos-14.

### ISSUE THREAD

**Issue by mbotsu** [xavierLowmiller/xcodegen-action#12]
Hi.

My issue is resolved, but I would like to provide information for anyone who may have encountered a similar issue.

I had permission issues when using `macos-14`.
As mentioned in the linked post, in advance.
You need to run `sudo chown runner:admin /usr/local/share`.

https://github.com/actions/runner-images/issues/9272#issuecomment-1922721091

```
Run xavierLowmiller/xcodegen-action@1.2.0
/usr/bin/curl --silent --create-dirs --location --output /Users/runner/action-xcodegen/xcodegen.zip https://github.com/yonaskolb/XcodeGen/releases/download/2.39.1/xcodegen.zip
/usr/bin/unzip -q -o /Users/runner/action-xcodegen/xcodegen.zip
/bin/rm /Users/runner/action-xcodegen/xcodegen.zip
/Users/runner/action-xcodegen/xcodegen/install.sh
cp: /Users/runner/action-xcodegen/xcodegen/share: unable to copy extended attributes to /usr/local/share: Bad file descriptor
cp: /usr/local/share/xcodegen: Permission denied
```

**Comment by bryce-b** [comment]
+1

**Comment by xavierLowmiller** [comment]
@mbotsu could you check if the issue is resolved after the release of #13 and close this?
